docs/dev/table/connectors/formats/debezium.md

```diff
@@ -154,6 +162,50 @@ SELECT * FROM topic_products;
 Format Options
 ----------------
 
+Flink provides `debezium-avro-confluent` and `debezium-json` formats to interpret Avro or Json messages produced by Debezium.
+Please pick corresponding format according to your Debezium message encoding.
+
```

Comment:
Use format `debezium-avro-confluent` to interpret Debezium Avro messages and format `debezium-json` to interpret Debezium Json messages.